provider/datagen/src/cldr/mod.rs

```diff
@@ -8,9 +8,8 @@
 //!
 //! This crate contains two implementations of [`CldrPaths`]:
 //!
-//! - [`CldrPathsLocal`] points to local copies of the CLDR JSON repositories.
-//! - [`CldrAllInOneDownloader`](download::CldrAllInOneDownloader) downloads and caches the
-//!   CLDR JSON repositories. Requires the "download" feature.
+//! - [`CldrPathsLocal`] which points directly to each local CLDR file, and
+//! - [`CldrPathsAllInOne`] which points to a local CLDR tree.
```

Comment:
Do we need both of these going forward? `CldrPathsLocal` is unused and I can't think of a scenario where a client has CLDR data but it's not in the CLDR tree.

Reply:
If getting rid of `CldrPathsLocal` makes your life easier, you can get rid of it.

Reply:
It doesn't make it harder right now, I was just wondering what the idea behind it was.

Reply:
Versions of CLDR prior to 38 or 39 had separate repos and zip files for each tree, but then they merged them into one big tree. I kept the code for the old layout for the time being, but it can probably be removed now.